lib/options/dcu.robot

```diff
@@ -0,0 +1,29 @@
+*** Settings ***
+Documentation       Library for UEFI configuration using Dasharo Configuration
+...                 Utility tool. Commonly used when serial port is not
+...                 available.
+
+Library             Collections
+Library             OperatingSystem
+Library             String
+
+
+*** Keywords ***
+Set UEFI Option
+    [Documentation]    Set an UEFI option to a value.
+    [Arguments]    ${option_name}    ${value}
+    Run    git clone https://github.com/Dasharo/dcu
+    # TODO: Remove once smmstore support is merged
+    Run    cd dcu && git checkout smmstore > /dev/null 2>&1 && cd ..
+    Power On
```

Comment:
Do we still need power control, even though we use SSH, not serial?
Can we imagine a scenario with no power control - e.g. laptop freestanding on my desk, just connected to the network?
If we need `Power On`, we still need RTE, right?

Reply:
a Sonoff should be enough, as long as `Power on AC` is enabled

Reply:
Changed to reboot in commit 563fb1e655de14fca030709837f7edf40a603157 . As a side effect the tests take half as much time now :)